packages/plugins/GoPlusSecurity/src/SNSAdaptor/index.tsx

```diff
@@ -3,26 +3,26 @@ import { base } from '../base'
 import { CheckSecurityDialog } from './CheckSecurityDialog'
 import { useState } from 'react'
 import { ApplicationEntry } from '@masknet/shared'
+import { SecurityCheckerIcon } from '@masknet/icons'
 
 const sns: Plugin.SNSAdaptor.Definition = {
     ...base,
     init(signal) {},
     ApplicationEntries: [
         {
-            RenderEntryComponent() {
+            RenderEntryComponent({ disabled, icon, title }) {
                 const [open, setOpen] = useState(false)
                 return (
                     <>
-                        <ApplicationEntry
-                            title="Check Security"
-                            icon={new URL('../assets/security-icon.png', import.meta.url).toString()}
-                            onClick={() => setOpen(true)}
-                        />
+                        <ApplicationEntry title={title} disabled={disabled} icon={icon} onClick={() => setOpen(true)} />
                         {open && <CheckSecurityDialog open={open} onClose={() => setOpen(false)} />}
                     </>
                 )
             },
-            defaultSortingPriority: 12,
+            name: { i18nKey: 'plugin_name', fallback: 'Check Security' },
```

Comment:
please rename this key to `__plugin_name`. otherwise, it will emit a warning.

Reply:
done